packages/dockest/src/utils/createDefaultReadinessChecks.ts

```diff
@@ -0,0 +1,40 @@
+import { execaWrapper } from './execaWrapper'
+import { DefaultReadinessChecks, Runner } from '../@types'
+
+export const createDefaultReadinessChecks = ({
+  runner,
+  runner: {
+    containerId,
+    dockerComposeFileService: { ports },
+  },
+}: {
+  runner: Runner
+}): DefaultReadinessChecks => ({
+  postgres: async ({ database, username }: { database: string; username: string }) => {
+    // Ref: http://postgresguide.com/utilities/psql.html
+    const command = `docker exec ${containerId} bash -c " \
+                        psql \
+                        -h localhost \
+                        -d ${database} \
+                        -U ${username} \
+                        -c 'select 1'"`
+
+    await execaWrapper(command, { runner })
+  },
+
+  redis: async () => {
+    const command = `docker exec ${containerId} redis-cli \
+                        -h localhost \
+                        -p ${ports[0].target} \
```

Comment:
Might be out of scope here: https://github.com/erikengervall/dockest/issues/143#issuecomment-590751259

Reply:
I had some additional [thoughts](https://github.com/erikengervall/dockest/issues/143#issuecomment-592666500) on that issue so will not include changes relating to it in this PR.